docs/cloud/pouta/tutorials/image-pipeline.md

```diff
@@ -0,0 +1,650 @@
+
+We go back to `terminal_C`, where we are still logged in to our virtual machine in cPouta, and we follow the [instructions on how to install and configure s3cmd](../../../data/Allas/using_allas/s3_client.md#getting-started-with-s3cmd).
+It basically boils down to running the following commands:
+```
+$ sudo apt update
+
+$ sudo apt install python3-openstackclient s3cmd
+
+$ wget https://raw.githubusercontent.com/CSCfi/allas-cli-utils/master/allas_conf
+
+$ source allas_conf --mode S3 --user your-csc-username
+```
+
+Once these three steps are done, we test that everything works properly by listing our buckets in Allas:
+```
+$ s3cmd ls
+2025-01-16 14:59  s3://input_bucket
+2025-01-16 14:59  s3://output_bucket
+```
+We notice that we can see the two buckets that we have created earlier in the tutorial.
+We have thus confirmed that the virtual machine can now access properly the buckets in Allas.
+
+### Configuring access from virtual machine to database
+
+At this point in the tutorial, traffic from the virtual machine is allowed to flow towards the database in Pukki.
+However, at the moment the virtual machine has no information about where the database can be found, i.e., its public IP address, nor which are the credentials to use when accessing the database.
+Therefore, next we configure the access to the database from the virtual machine.
+
+We stay on `terminal_C`.
+First, we install a tool that we will need to talk with the database:
+```
+$ sudo apt install postgresql-client
+Reading package lists... Done
+Building dependency tree... Done
+Reading state information... Done
+The following NEW packages will be installed:
+  postgresql-client
+0 upgraded, 1 newly installed, 0 to remove and 28 not upgraded.
+Need to get 11.6 kB of archives.
+After this operation, 17.4 kB of additional disk space will be used.
+Get:1 http://nova.clouds.archive.ubuntu.com/ubuntu noble-updates/main amd64 postgresql-client all 16+257build1.1 [11.6 kB]
+Fetched 11.6 kB in 0s (47.3 kB/s)             
+Selecting previously unselected package postgresql-client.
+(Reading database ... 117863 files and directories currently installed.)
+Preparing to unpack .../postgresql-client_16+257build1.1_all.deb ...
+Unpacking postgresql-client (16+257build1.1) ...
+Setting up postgresql-client (16+257build1.1) ...
+Scanning processes...                                                                                                                                                                    
+Scanning linux images...                                                                                                                                                                 
+
+Running kernel seems to be up-to-date.
+
+No services need to be restarted.
+
+No containers need to be restarted.
+
+No user sessions are running outdated binaries.
+
+No VM guests are running outdated hypervisor (qemu) binaries on this host.
+```
```

Comment:
IMO Install postgresql-client 

```
sudo apt update ; sudo apt install postgresql-client
```

should be enough

Reply:
Fixed in cdd97869bd1a29e958b9b243d02f2911e979faa4